Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -378,15 +427,194 @@ def run_batch_admin_cmd(host_ids: list, command_type: str, full_command: str):
         :return: Response JSON which contains errors (if exist) and retrieved resources
     """
     endpoint_url = '/real-time-response/combined/batch-admin-command/v1'
-    batch_id = init_rtr_session(host_ids)
+    batch_id = init_rtr_batch_session(host_ids)
 
     body = json.dumps({
         'base_command': command_type,
         'batch_id': batch_id,
         'command_string': full_command
     })
     response = http_request('POST', endpoint_url, data=body)
+    return response
+
+
+def run_batch_get_cmd(host_ids: list, file_path:str, optional_hosts:list = None, timeout:int = None, timeout_duration:str = None) -> Dict:
+    """
+        Batch executes `get` command across hosts to retrieve files.
+        After this call is made `/real-time-response/combined/get-command-status/v1` is used to query for the results.
+
+      :param host_ids: List of host agent ID’s to run RTR command on.
+      :param file_path: Full path to the file that is to be retrieved from each host in the batch.
+      :param optional_hosts: List of a subset of hosts we want to run the command on. If this list is supplied, only these hosts will receive the command.
+      :param timeout: Timeout for how long to wait for the request in seconds
+      :param timeout_duration: Timeout duration for for how long to wait for the request in duration syntax
+    """
+    endpoint_url = '/real-time-response/combined/batch-get-command/v1'
+    batch_id = init_rtr_batch_session(host_ids)
+
+    body = {
+        'batch_id': batch_id,
+        'file_path': file_path
+    }
+    if optional_hosts:
+        body['optional_hosts'] = optional_hosts
 
+    params = {k:v for k,v in {
```

Comment:
not sure why this is needed, can just intialize a dict

Reply:
Will change to
`
params = {}
if timeoutNone:
    params['timeout'] = timeout
if timeout_durationNone:
    params['timeout_duration'] = timeout_duration
`
for better readable.

Reply:
you can use `assign_params()` for that